firefox-ios/firefox-ios-tests/Tests/ClientTests/ShoppingProductTests.swift

```diff
@@ -313,6 +313,18 @@ final class ShoppingProductTests: XCTestCase {
         let result = sut.supportedTLDWebsites
         XCTAssertNil(result)
     }
+
+    func testReportAdEvent_WithValidURL_CallsClientAPI() async throws {
+        let url = URL(string: "https://www.amazon.com/Under-Armour-Charged-Assert-Running/dp/B087T8Q2C4")!
+        let sut = ShoppingProduct(url: url, client: client)
+
+        _ = try await sut.reportAdEvent(eventName: "click", eventSource: "web", aidvs: ["aidv1", "aidv2"])
```

Comment:
Is this testing against a dummy endpoint? Otherwise the tests are influencing the metrics.

Reply:
Yes, api client is mocked.